generators/generator-base.js

```diff
@@ -1531,6 +1531,29 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
     return this.getExistingEntities().map(entity => entity.name);
   }
 
+  /**
+   * Read entity json from config folder.
+   * @param {string} entityName - Entity name
+   * @return {object} entity definition
+   */
+  readEntityJson(entityName) {
+    const configDir = this.destinationPath(JHIPSTER_CONFIG_DIR);
+    const file = this.destinationPath(configDir, `${entityName}.json`);
+    try {
+      const definition = this.fs.readJSON(file);
+      if (this.options.namespace !== 'jhipster:info') {
+        // Execute a write operation to set the file as modified on mem-fs to trigger prettier.
+        this.fs.append(file, '', { trimEnd: false, separator: '' });
+      }
+      return definition;
+    } catch (error) {
+      // not an entity file / malformed?
```

Comment:
good comment, but what do you want to do about that?
keeping the comment won't do anything because it will be forgotten

what about adding something more to the warning? like `unable to parse blablabla, is the entity file malformed or invalid?`?